server/athenian/api/models/state/versions/81fe0d1e6cc7_goaltemplate.py

```diff
@@ -0,0 +1,74 @@
+"""GoalTemplate
+
+Revision ID: 81fe0d1e6cc7
+Revises: bd6c6cedf626
+Create Date: 2022-08-09 15:27:53.818524+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "81fe0d1e6cc7"
+down_revision = "bd6c6cedf626"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "goal_template_account_inits",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.func.now(),
+            nullable=False,
+        ),
+        sa.Column("account_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["account_id"],
+            ["accounts.id"],
+            name="fk_goal_template_account_init_account",
+            ondelete="CASCADE",
+        ),
+        sa.PrimaryKeyConstraint("account_id"),
+    )
+    op.create_table(
+        "goal_templates",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.func.now(),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.func.now(),
+            nullable=False,
+        ),
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("account_id", sa.Integer(), nullable=False),
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column("metric", sa.String(), nullable=False),
+        sa.ForeignKeyConstraint(["account_id"], ["accounts.id"], name="fk_goal_template_account"),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("account_id", "name", name="uc_goal_templates_account_id_name"),
+        sqlite_autoincrement=True,
+    )
+
```

Comment:
We should insert predefined template for existing accounts directly in the migration.

Reply:
The `migrate_goal_templates.py` script does this and also change template_id for existing goals to point to DB templates.
This way we can deploy this code now and execute the script after frontend is using the templates API, after that we'll use and serve template IDs from DB instead of hard-coded IDs